### DISCUSSION TITLE
is Encryption working on Version: v0.107.44?

### DISCUSSION THREAD

**Discussion by udtcp** [AdguardTeam/AdGuardHome#6732] [Q&A]
Running AGH on openwrt MT6000 router
I have DOH in the Upstream DNS servers
I choose enable encryption and disable enable plain dns and save. But after logging in and out the tick is back on plian dns and untick in encryption option. Do i need to do anything else or could it be a bug?

**Comment by udtcp** [comment]
![encryption](https://github.com/AdguardTeam/AdGuardHome/assets/154565507/cd8058b8-cd3d-4fac-96c1-5596314f262f)

**Reply by udtcp** [reply]
Does the encryption box only works as enabled if certificates are installed?

**Reply by amrmzr** [reply]
Yes it's working..

**Reply by Matt39823049823** [reply]
I have the same issue.  DNS servers have HTTPS, QUIC server addresses that test as working.  WITH the 'Enable plain DNS' checked DNS requests flow fine.  With that UNCHECKED, zero DNS requests are being passed.
Have the following firewall ports allowed:
 ports: 3000/tcp 53/udp 53/tcp 80/tcp 22/tcp 443/tcp 853/tcp 853/udp
 Can supply any logs or AdGuardHome.yaml files if needed.

**Reply by amrmzr** [reply]
You're right. I tried unchecking "Enable Plain DNS" and it stopped sending & receiving queries as you said.
